djangocms_googlemap/models.py

```diff
@@ -1,90 +1,287 @@
+    # lowercase of the class name of this model.
+    # https://github.com/divio/django-cms/issues/5030
+    cmsplugin_ptr = models.OneToOneField(
+        CMSPlugin,
+        related_name='%(app_label)s_%(class)s',
+        parent_link=True,
+    )
 
-    height = models.CharField(
-        _('height'), max_length=6, default='400px',
-        help_text=_('Plugin height (in pixels).'))
+    def __str__(self):
+        if self.title:
+            return self.title
+        return str(self.pk)
 
-    info_window = models.BooleanField(
-        _('info window'), default=True,
-        help_text=_('Show textbox over marker'))
+    def get_short_description(self):
+        display = ''
+        if self.title:
+            display = '{0}, '.format(self.title)
+        elif self.lat and self.lng:
+            display += '{0} / {1}, '.format(self.lat, self.lng)
+        display += '{0} x {1}'.format(self.width, self.height)
+        return display
 
-    scrollwheel = models.BooleanField(
-        _('scrollwheel'), default=True,
-        help_text=_('Enable scrollwheel zooming on the map'))
+    def clean(self):
+        if self.width and not CSS_WIDTH_RE.match(self.width):
+            raise ValidationError(
+                _('Width must be a positive integer followed by "px" or "%".')
+            )
+        if self.height and not CSS_HEIGHT_RE.match(self.height):
+            raise ValidationError(
+                _('Height must be a positive integer followed by "px".')
+            )
+        if self.style:
+            try:
+                json.loads(self.style)
+            except ValueError:
+                raise ValidationError(_('Map styling has to be valid JSON.'))
 
-    double_click_zoom = models.BooleanField(
-        _('double click zoom'), default=True)
 
-    draggable = models.BooleanField(_('draggable'), default=True)
+@python_2_unicode_compatible
+class GoogleMapMarker(CMSPlugin):
+    """
+    Renders a marker inside the Google Maps wrapper
+    """
+    title = models.CharField(
+        verbose_name=_('Title'),
+        max_length=255,
+        blank=True,
+    )
+    address = models.CharField(
+        verbose_name=_('Full address'),
+        max_length=255,
+        blank=True,
+        help_text=_('Latitude and longitude can be used to fine tune the location.'),
+    )
+    lat = models.DecimalField(
+        verbose_name=_('Latitude (lat)'),
+        max_digits=10,
+        decimal_places=6,
+        null=True,
+        blank=True,
+        help_text=_('GPS latitute (e.g. "46.947973").'),
+    )
+    lng = models.DecimalField(
+        verbose_name=_('Longitude (lng)'),
+        max_digits=10,
+        decimal_places=6,
+        null=True,
+        blank=True,
+        help_text=_('GPS longitude (e.g. "7.447446").'),
+    )
+    # info window settings
+    show_content = models.BooleanField(
+        verbose_name=_('Show window'),
+        default=True,
+    )
```

Comment:
Shows the info window on load, so it is displayed per default:

![image](https://cloud.githubusercontent.com/assets/270307/19931008/34cd4b78-a10b-11e6-9841-67bba26bfa07.png)

If not selected, will show on click (on the pin)

You can enable/disable it.